packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -58,10 +58,22 @@ interface Props {
 
 const AppNavigation = ({ items }: Props) => {
     const routeName = useSelector(state => state.router.route?.name);
+    const routerParams = useSelector(state => state.router.params);
```

Comment:
you can modify first `useSelector` condition and use `router` const instead of fetching same reducer twice

Reply:
in fact this fetch is not necessary at all, if you use `selectedAccount.params` in `onClick` block